pkg/scheduler/queue/query_component_utilization.go

```diff
@@ -97,65 +72,6 @@ func NewQueryComponentUtilization(
 	}, nil
 }
 
-// ExceedsThresholdForComponentName checks whether a query component has exceeded the capacity utilization threshold.
```

Comment:
this was all for when we were going to use this for the load-balancing algorithm